### ISSUE TITLE
Keeps asking to be copied into Applications folder

### ISSUE THREAD

**Issue by noelcosgrave** [PostgresApp/PostgresApp#320]
The app keeps asking if I want to copy it to the applications folder even though it has already been copied there and is being run from Applications. My Applications folder is a symlink, so that might be a contributory factor.

**Comment by jakob** [comment]
Did you really replace /Applications with a symlink? Doesn't that cause lots of trouble? I'm curious why you'd want to do that!

Anyway, you can check out the source from Github and make a DEBUG build. Postgres.app doesn't check the location in DEBUG builds.

**Comment by noelcosgrave** [comment]
Yes, I did replace it with a symlink and no, it hasn't caused any trouble, not so far. My boot disk is a 1TB SSD, but its not enough, so I symlinked Applications to much larger spinning-rust disk. I have a lot of audio and video apps that each come complete with 100s of gigs of libraries, not to mention everything that comes with Adobe CC, so it was done out of necessity!

See http://osxdaily.com/2015/08/06/make-symbolic-links-command-line-mac-os-x/

Thanks! I'll try out the debug build!
